### ISSUE TITLE
no-as-type-assertion false positive for const assertions (ts 3.4) 

### ISSUE THREAD

**Issue by WiseBird** [ajafff/tslint-consistent-codestyle#146]
In typescript 3.4 new syntax was introduced - [const assertions](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions)

```
// Type '"hello"'
let x = "hello" as const;

// Type 'readonly [10, 20]'
let y = [10, 20] as const;

// Type '{ readonly text: "hello" }'
let z = { text: "hello" } as const;
```

no-as-type-assertion highlights this usages as errors

**Comment by ajafff** [comment]
That's because you also write it as `<const>"hello"`
